Defuq/Assets/Editor/FovEditor.cs

```diff
@@ -2,41 +2,39 @@
 using System.Collections.Generic;
 using UnityEngine;
 using UnityEditor;
+using UI.Monitors;
 
-namespace amitScripts
+[CustomEditor(typeof(EnemyBehavior))]
+public class FovEditor : Editor
 {
-    [CustomEditor(typeof(EnemyBehavior))]
-    public class FovEditor : Editor
+    private void OnSceneGUI()
     {
-        private void OnSceneGUI()
-        {
-            // Circle Fow
-            EnemyBehavior fow = (EnemyBehavior)target;
-            Handles.color = Color.white;
-            Handles.DrawWireArc(fow.transform.position, Vector3.up, Vector3.forward, 360, fow.currentRadius);
-            // Circle AttackRng
-            Handles.color = Color.red;
-            Handles.DrawWireArc(fow.transform.position, Vector3.up, Vector3.forward, 360, fow.currentAttackRange);
+        // Circle Fow
+        EnemyBehavior fow = (EnemyBehavior)target;
```

Comment:
fow? name unclear. Please choose a better name that better explain what this object dose